src/args/argument_list_test.go

```diff
@@ -81,27 +89,25 @@ func Test_ArgumentList_CreateAPIConfig(t *testing.T) {
 				Port:                   "8500",
 				Token:                  "my_token",
 				EnableSSL:              true,
-				TrustServerCertificate: false,
-				CABundleDir:            "ca_dir",
-				CABundleFile:           "ca_file",
+				TrustServerCertificate: true,
```

Comment:
would make sense to keep this case and add an extra with this `TrustServerCertificate: true` ?